react/src/Datasets/components/AnalysisRunnerDialog.tsx

```diff
@@ -67,27 +74,61 @@ export default function AnalysisRunnerDialog({
                     Run a pipeline using the selected datasets. A full analysis can take a day to
                     several days depending on the number of datasets and the requested pipeline.
                 </Typography>
-                <FormControl component="fieldset">
-                    <FormLabel component="legend" className={classes.text}>
-                        Pipelines:
-                    </FormLabel>
-                    <RadioGroup
-                        row
-                        aria-label="Pipelines"
-                        name="pipelines"
-                        value={pipeline}
-                        onChange={event => setPipeline(parseInt(event.target.value))}
-                    >
-                        {pipelines.map(({ pipeline_id, pipeline_name, pipeline_version }) => (
-                            <FormControlLabel
-                                key={pipeline_id}
-                                label={`${pipeline_name} ${pipeline_version}`}
-                                value={pipeline_id}
-                                control={<Radio color="primary" />}
-                            />
-                        ))}
-                    </RadioGroup>
-                </FormControl>
+                <Grid container direction="column">
+                    <Grid item>
+                        <FormControl component="fieldset">
+                            <FormLabel
+                                component="legend"
+                                className={pipeline ? classes.text : classes.warningText}
+                            >
+                                Pipelines*:
+                            </FormLabel>
```

Comment:
What's the reason for distinguishing this label this way?

Reply:
I was trying to indicate that the priority field is required by adding the asterisk to the label, coloring the label red if the value is missing, and disabling the button if the value is missing. I can take another approach or remove if this is undesirable!

Reply:
It's not really indicated locally what this means. I don't think that this is really needed.

Reply:
removed in [1cde926](https://github.com/ccmbioinfo/stager/pull/522/commits/1cde9269aa0eddce800c5605a045ad99a9999b30)